### PR TITLE
[DS] Typography in webhooks

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What

Integrated Typography in Webhooks views
Fixed snapshots previously not waiting for data to be loaded 
Fixed Combobox HeadersInput not accepting newly created value as ComboboxOption

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/EventInput/index.js [hunk 1]
```diff
@@ -121,14 +121,18 @@ const EventInput = ({ isDraftAndPublish }) => {
                         'This event only exists for contents with Draft/Publish system enabled',
                     })}
                   >
-                    <TableLabel textColor="neutral600">{formatMessage({ id: header })}</TableLabel>
+                    <Typography variant="sigma" textColor="neutral600">
+                      {formatMessage({ id: header })}
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/EventInput/index.js:125]:
```js
formatMessage({ id: header, defaultMessage: header })
```

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/EventInput/index.js [hunk 2]
```diff
@@ -121,14 +121,18 @@ const EventInput = ({ isDraftAndPublish }) => {
...
+                    </Typography>
                   </td>
                 );
               }
 
               return (
                 <td key={header}>
-                  <TableLabel textColor="neutral600">{formatMessage({ id: header })}</TableLabel>
+                  <Typography variant="sigma" textColor="neutral600">
+                    {formatMessage({ id: header })}
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/EventInput/index.js:134]: Same here.

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/TriggerContainer/index.js [hunk 3]
```diff
@@ -75,29 +76,24 @@ const Message = ({ statusCode, message }) => {
...
-            whiteSpace: 'nowrap',
-            textOverflow: 'ellipsis',
-          }}
-        >
-          {message}
-        </Text>
+      <Flex justifyContent="flex-end">
+        <Flex maxWidth={pxToRem(250)} justifyContent="flex-end" title={message}>
+          <Typography ellipsis textColor="neutral600">
+            {message}YOLOOOOOOdjkeb bdewib dehwb dilwb dilhwebd ilewb dlw
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/TriggerContainer/index.js:94]: lol  😂

**Review by soupette — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/TriggerContainer/index.js (hunk 3 above)
Comment (derrickmehaffy) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/TriggerContainer/index.js:94]: :laughing:

Comment (ronronscelestes) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/TriggerContainer/index.js:94]: I think we should leave it that way 🤓

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**